pkg/arvo/app/http-server-store.hoon

```diff
@@ -0,0 +1,220 @@
+/-  *http-server-store
+/+  *server, default-agent, verb, dbug
+|%
++$  card  card:agent:gall
++$  versioned-state
+  $%  state-zero
+  ==
+::
++$  serving  (map url-base=path [clay-base=path public=?])
++$  state-zero
+  $:  %0
+      =configuration
+      =serving
+  ==
+--
+%+  verb  |
+%-  agent:dbug
+=|  state-zero
+=*  state  -
+^-  agent:gall
+|_  =bowl:gall
++*  this  .
+    def   ~(. (default-agent this %|) bowl)
+::
+++  on-init
+  ^-  (quip card _this)
+  |^
+  :_  %_  this
+          serving
+        %-  ~(gas by *^serving)
+        %+  turn
+          ^-  (list path)
+          [/ /'~landscape' /'~chat' /'~link' /'~dojo' /'~groups' /'~publish' ~]
+        |=(pax=path [pax [/app/landscape %.n]])
+      ==
+  :~  (connect /)
+      (connect /'~landscape')
+      (connect /'~chat')
+      (connect /'~link')
+      (connect /'~dojo')
+      (connect /'~groups')
+      (connect /'~publish')
+  ==
```

Comment:
If you're serving the root path, why do you also need to serve these other paths. Also doesn't this contradict the earlier conversation we had about not hard-coding these?

Reply:
They aren't hard-coded. They're dynamically registered in the state just like any other path would be and can be unbound at any time. Imagine a case in which you wanted to unbind the / path from the %http-server-store to use in some other context. If you don't already have the specific app paths bound, then suddenly everything breaks. If you do have the specific app paths bound, then everything still works as normal. Also, the `get-suffix` arm you had didn't work to serve the SPA until I made this change, so it isn't just theoretical...

Reply:
Surely "not hard-coded" would involve, even in the initialization logic, pulling from the filesystem or sth rather than having an explicit list in the source code twice?

Reply:
Okay, I understood something else by "dynamicity", I thought you wanted to have each app poke this one. This seems fine.

Reply:
Mark is right, yeah. Will move them to the individual apps.